src/Model/GroundWaterFlow/gwf-ghba.f90

```diff
@@ -0,0 +1,467 @@
+        call store_error(errmsg)
+      end if
+    end do
+    !
+    !write summary of ghb package error messages
+    if (count_errors() > 0) then
+      call store_error_unit(this%inunit)
+    end if
+  end subroutine ghba_ck
+
+  !> @brief Formulate the HCOF and RHS terms
+  !!
+  !! Skip if no GHBs
+  !<
+  subroutine ghba_cf(this)
+    ! -- dummy
+    class(GhbaType) :: this
+    ! -- local
+    integer(I4B) :: i, noder
+    !
+    do i = 1, this%nbound
+      noder = this%nodelist(i)
+      if (noder == 0) cycle
+      if (this%ibound(noder) .le. 0) then
+        this%hcof(i) = DZERO
+        this%rhs(i) = DZERO
+        cycle
+      end if
+      this%hcof(i) = -this%cond_mult(i)
+      this%rhs(i) = -this%cond_mult(i) * this%bhead(i)
+    end do
+  end subroutine ghba_cf
+
+  !> @brief Copy rhs and hcof into solution rhs and amat
+  !<
+  subroutine ghba_fc(this, rhs, ia, idxglo, matrix_sln)
+    ! -- dummy
+    class(GhbaType) :: this
+    real(DP), dimension(:), intent(inout) :: rhs
+    integer(I4B), dimension(:), intent(in) :: ia
+    integer(I4B), dimension(:), intent(in) :: idxglo
+    class(MatrixBaseType), pointer :: matrix_sln
+    ! -- local
+    integer(I4B) :: i, noder, ipos
+    real(DP) :: cond, bhead, qghba
+    !
+    ! -- pakmvrobj fc
+    if (this%imover == 1) then
+      call this%pakmvrobj%fc()
+    end if
+
+    do i = 1, this%nbound
+      noder = this%nodelist(i)
+      if (noder == 0) cycle
+      rhs(noder) = rhs(noder) + this%rhs(i)
+      ipos = ia(noder)
+      call matrix_sln%add_value_pos(idxglo(ipos), this%hcof(i))
+      !
+      ! -- If mover is active and this boundary is discharging,
+      !    store available water (as positive value).
+      bhead = this%bhead(i)
+      if (this%imover == 1 .and. this%xnew(noder) > bhead) then
+        cond = this%cond_mult(i)
+        qghba = cond * (this%xnew(noder) - bhead)
+        call this%pakmvrobj%accumulate_qformvr(i, qghba)
+      end if
+    end do
+  end subroutine ghba_fc
+
+  !> @brief Define the list heading that is written to iout when PRINT_INPUT
+  !! option is used
+  !<
+  subroutine define_listlabel(this)
+    ! -- dummy
+    class(GhbaType), intent(inout) :: this
+    !
+    ! -- create the header list label
+    this%listlabel = trim(this%filtyp)//' NO.'
+    if (this%dis%ndim == 3) then
+      write (this%listlabel, '(a, a7)') trim(this%listlabel), 'LAYER'
+      write (this%listlabel, '(a, a7)') trim(this%listlabel), 'ROW'
+      write (this%listlabel, '(a, a7)') trim(this%listlabel), 'COL'
+    elseif (this%dis%ndim == 2) then
+      write (this%listlabel, '(a, a7)') trim(this%listlabel), 'LAYER'
+      write (this%listlabel, '(a, a7)') trim(this%listlabel), 'CELL2D'
+    else
+      write (this%listlabel, '(a, a7)') trim(this%listlabel), 'NODE'
+    end if
+    write (this%listlabel, '(a, a16)') trim(this%listlabel), 'STAGE'
+    write (this%listlabel, '(a, a16)') trim(this%listlabel), 'CONDUCTANCE'
+    if (this%inamedbound == 1) then
```

Comment:
Is bound name an option?  I think not.  No need to do anything here.  Just asking out of curiosity.

Reply:
It isn't an option- I can remove this as it will probably be confusing that it's there.